doc/api/process.md

```diff
@@ -1103,6 +1103,25 @@ and [Cluster][] documentation), the `process.connected` property will return
 Once `process.connected` is `false`, it is no longer possible to send messages
 over the IPC channel using `process.send()`.
 
+## `process.constrainedMemory()`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+Gets the amount of memory available to the process (in bytes) based on
+limits imposed by the OS. If there is no such constraint, or the constraint
+is unknown, `0` is returned.
```

Comment:
Why does it return `0` in this case? Why not `null` or `undefined`?

Reply:
Just return the value from Libuv API currently.

Reply:
That doesn't seem like a particularly safe design IMHO. It's very C-ish, but not very JS-ish.

Reply:
It's not a blocking concern for me but I agree with @tniessen

Reply:
Thanks, i have updated the code that the API will return `undefined` if the value is `0`.